addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -401,6 +401,52 @@ var FieldMany2One = AbstractField.extend({
             }
         };
     },
+    /**
+     * @private
+     * @param {Object} values
+     * @param {string} search_val
+     * @param {Object} domain
+     * @param {Object} context
+     * @returns {Object}
+     */
+    _manageSearchMore: function (values, search_val, domain, context) {
```

Comment:
@hbh-odoo why not passing option, something like ```no_search_more```, like we have ```no_open```, ```no_create_edit```, with this we don't need to override many2one field for this feature

Reply:
@msh-odoo Yes! I've already suggested that but @VincentSchippefilt don't want to add one more option for m2o as the number of options for the many2one is already large.

Reply:
@msh-odoo indeed, this will be used only once and temporarily, I don't want to add options for that reason.